Project.toml

```diff
@@ -47,6 +48,7 @@ Graphs = "1"
 LinearAlgebra = "1.9"
 MMTF = "1"
 MetaGraphs = "0.7, 0.8"
+MsgPack = "1.2.1"
```

Comment:
Would be good to know how much time this adds to `using BioStructures`. I expect not a lot, but it can be put in an extension if it is significant.